src/main/resources/caseIdForNotifyRpaOnCaseHandedOffline.txt

```diff
@@ -1,10 +1,4 @@
-1671554345936878
-1671719628161672
-1673866074871359
-1674214750411548
-1675856602677905
-1676393623339520
-1656068208985525
+1671554345936878,1671719628161672,1673866074871359,1674214750411548,1675856602677905,16763936233395201656068208985525
```

Comment:
Can you roll back the changes, I don't think it is related

Reply:
oh sure.

Reply:
done